src/test/java/com/conveyal/gtfs/TestUtils.java

```diff
@@ -35,7 +35,11 @@ public static void dropDB(String dbName) {
             "AND pid <> pg_backend_pid()", dbName
         ));
         // drop the db
-        executeAndClose(String.format("DROP DATABASE %s", dbName));
+        if(executeAndClose(String.format("DROP DATABASE %s", dbName))) {
```

Comment:
Something I just thought of: what if someone has a database already for the DB name that gets auto-generated?  The tests won't try to use that and/or drop it upon completion, will they?  Just making sure...

Reply:
hmm, well the tests would run into an error because they would try creating a database that already existed.  But yeah, it's a good idea to generate more random database name.